docs/docs/projects/README.md

```diff
@@ -77,8 +79,72 @@ At this time it's not possible to move workflows out of GitHub Synchronized Proj
 
 :::
 
+## Access permissions
+
+### Understanding who has access to which project
+We've introduced 2 new columns in the [project list view](https://pipedream.com/projects): **Owner** and **Access**.
+
+**Owner** indicates who within the workspace owns each project. This is typically the person who created the project.
+
+![Project Listing (Owner)](./images/project-listing-owner.png)
+
+::: warning Some projects may not have owners
+Projects created before February 2024 don't automatically have owners.
+:::
+
+**Access** indicates who within the workspace has access to each project, and this can be displayed as "me", "Workspace", or "N members".
```

Comment:
```suggestion
**Access** indicates which workspace members have access to each project, and this can be displayed as "me", "Workspace", or "N members".
```

Just switching to an active voice and simplifying the subject of the sentence.